src/terminal/SixelParser_test.cpp

```diff
@@ -338,3 +338,17 @@ TEST_CASE("SixelParser.newline", "[sixel]")
         }
     }
 }
+
+TEST_CASE("SixelParser.vertical_cursor_advance", "[sixel]")
+{
+    auto constexpr defaultColor = RGBAColor { 0, 0, 0, 255 };
+    SixelImageBuilder ib(
+        { Width(5), Height { 30 } }, 1, 1, defaultColor, std::make_shared<SixelColorPalette>(16, 256));
```

Comment:
Maybe a copy'n'paste thing, but there's a style breakage. Width is using `()` whereas Height is using `{}`. both solution work, but maybe only one should be used for consistency.